website/docs/r/cr_retention_policy.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "ibm"
```

Comment:
Add subcategory and remove sidebar_current

Reply:
Done